Assets/Scripts/HoldObject.cs

```diff
@@ -28,6 +30,11 @@ void Update()
         CarryObject();
     }
 
+    public GameObject get_PickedUpObject()
+    {
+        return pickedUpObject;
+    }
```

Comment:
```suggestion
    public GameObject GetPickedUpObject()
    {
        return pickedUpObject;
    }
```
Let's have consistent code style throughout the project